src/content/docs/tutorial-diagnose-infra-bottlenecks/instrument-your-system.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Collect data about your system infrastructure
+metaDescription: Capture data about your system by installing agents, then view that data in New Relic. 
+---
+
+Like the bridges and roads of a dense city, your system infrastructure needs regular inspection and maintenance. What processes affect CPU and disk usage? What app behaviors typically correlate with failing hosts? The performance of your infrastructure can affect the performance of your APIs, services, and apps. A database failure can prevent your server from handling data, or a memory leak can bring an otherwise performant app to an abrupt halt. 
+
+## Objectives [#objectives]
+
+This tutorial series walks you through using infrastructure data to troubleshoot different incident scenarios. While New Relic offers a full suite integrations to monitor the Cloud, containerized environments, and various virtual machines, this series is specific to data reported with an infrastructure agent that does not have additional integrations.
+
+That said, this doc explains how you might fully instrument your system. You will:
+
+* Install the infrastructure agent with our guided install
+* Understand the guided install's recommendations so you can make an informed decision about instrumentation
+* Learn about infrastructure integrations 
+
+## Install the infrastructure agent  [#install]
+
+When you observe your infrastructure in context, you can respond to incidents with more clarity and direction, letting you move from reaction to proaction in your incident response.
+
+<Steps>
+<Step>
+    ### Start with our guided install  
+
+To install the infrastructure agent, we recommend using our [guided install](https://one.newrelic.com/nr1-core?state=a36f15a1-c5ce-528c-2ec2-8bcaacc40c03). Our guided install is a CLI installer that lets New Relic inspect, then make recommendations about additional instrumentation to your system. These recommendations are for our agents and any integrations relevant to your system. 
+        
+    </Step>
+    <Step>
+
+### Instrument your system with agents
+
+The fidelity of your data largely depends on what you've chosen to report to New Relic. It's the difference between purchasing a point-and-shoot camera that may capture the essence of a scene without the detail, or a professional DSLR camera that brings a scene back to life. 
```

Comment:
What's the difference? The more you report the better the "picture." We're all dummies! Spell it out for us :)